cmd/main.go

```diff
@@ -221,44 +222,22 @@ func initMC() {
 
 }
 
-func installAutoCompletion(ctx *cli.Context) {
-	if ctx.Bool("no-autocompletion") || ctx.GlobalBool("no-autocompletion") {
-		return
-	}
-
-	if globalQuiet || globalJSON || !terminal.IsTerminal(int(os.Stdout.Fd())) {
-		return
-	}
-
+func installAutoCompletion() {
 	if runtime.GOOS == "windows" {
+		console.Infoln("autocompletion feature is not available for this operating system")
 		return
 	}
 
-	if completeinstall.IsInstalled("mc") {
+	if completeinstall.IsInstalled(filepath.Base(os.Args[0])) || completeinstall.IsInstalled("mc") {
+		console.Infoln("autocompletion is already enabled in your '$SHELLRC'")
```

Comment:
Why not simply 'shell' ? I mean did you intend to have a variable substitution here '$SHELLRC'  ?

Reply:
That was AB's recommendation @vadmeste